src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -57,12 +57,13 @@ public Arm(int hexEncoderPort, ShuffleboardTab armTab) {
         motorSpeedWidget = armTab.add("Arm speed", 0).withWidget(BuiltInWidgets.kNumberBar)
                 .withPosition(3, 2).withSize(1, 1).getEntry();
         initialAngle = 0;
+        rotationController.setSetpoint(initialAngle);
     }
 
     @Override
     public void periodic() {
-        //double pid = getPIDController().calculate(getAngle(), targetAngle);
-        //setMotor(pid);
+        double pid = rotationController.calculate(getAngle());
```

Comment:
right now just using a PID controller, should we use feedforward with:
https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/controller/ArmFeedforward.html and https://docs.wpilib.org/en/stable/docs/software/advanced-controls/controllers/feedforward.html#armfeedforward
and probably look into ProfiledPIDController as well: 
https://docs.wpilib.org/en/stable/docs/software/advanced-controls/controllers/profiled-pidcontroller.html